src/main/java/core/basesyntax/service/impl/CsvReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.List;
+
+public class CsvReaderServiceImpl implements ReaderService {
+    private File file;
+
+    public CsvReaderServiceImpl(String filePath) {
+        file = new File(filePath);
+    }
+
+    @Override
+    public List<String> getLines() {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            return reader.lines().toList();
```

Comment:
if I'm not mistaken, reader.lines returns stream, you convert it to list, and then in Parser you work with IntStream:         ```
return IntStream.range(1, lines.size())
                .mapToObj(i -> lines.get(i).split(FIELD_SEPARATOR))
                .map(lineToOperation).toList();
```
You can work with stream, but filter the firstline to avoid a lot of converting